### ISSUE TITLE
Display notice on Settings > Payments > Deposits to inform the merchant when their bank account has an error.

### ISSUE THREAD

**Issue by jessy-p** [Automattic/woocommerce-payments#8331]
### Description
Display a notice in Settings > Payments > Deposits to inform the merchant when their bank account has an error.
- Based on https://github.com/Automattic/woocommerce-payments/issues/6294#issuecomment-1977424883
- Similar to the notice displayed on Payments > Overview > Deposits #8321 
- Dependent on woocommerce-payments-server#4862

### Acceptance criteria
When Stripe sets a bank account to `errored` after a failed deposit, the merchant is informed via a notice on  Settings > Payments > Deposits

### Designs
TBD

### Testing instructions
- Test with woocommerce-payments-server#4862
- Visit the Stripe Express dashboard for your test WooPayments account (Payments → Overview → Account Details → Edit Details) and update the payout bank account to a [failing test account](https://href.li/?https://docs.stripe.com/connect/testing#payouts), e.g. 110000000/000111111116
- Use the [Stripe Shell](https://href.li/?https://docs.stripe.com/connect/manual-payouts?shell=true&api=true&resource=accounts&action=retrieve) to create a manual deposit.
stripe payouts create --stripe-account="acct_123" --amount="123" --currency="USD" 
- The bank account would be set to `errored`.

### Dev notes
https://github.com/Automattic/woocommerce-payments/pull/8321
https://github.com/Automattic/woocommerce-payments-server/pull/4862

### Additional context
TBD: how to test for multiple currency --> bank account setups.

**Comment by jessy-p** [comment]
@elizaan36  https://github.com/Automattic/woocommerce-payments/issues/6294#issuecomment-1944434429

Please give input on design and priority.

**Comment by elizaan36** [comment]
@jessy-p What's the question?

**Comment by haszari** [comment]
Marking as `high`. I don't know how common this is but it seems important that if there is a bank error we show this in UI. 

Next steps:

- Test/explore to confirm that this is still a problem – i.e. if payout error, no clue in settings.
- Prototype / design
  - Engineer could implement something using standard patterns / components and/or
  - Design could mock up
- Implement, test, review ⇒ ship!

**Comment by haszari** [comment]
> Test/explore to confirm that this is still a problem – i.e. if payout error, no clue in settings.

@jessy-p when you have a moment, could you please:

- Recheck to confirm this is still a problem.
- Update the title and description to clarify how merchant is impacted, how to reproduce.

Then we can prioritise appropriately. Based on the info so far, and the fact that we have fixed some similar issues already (#6294, , I'm unclear if this is common or rare edge case.

**Comment by elizaan36** [comment]
👋 I'll have to pass the torch on this one to @rogermattic after today. I'm not clear on the design needs here, so as long as it's on your radar. Thanks Magda!

**Comment by rogermattic** [comment]
Hey @haszari and @jessy-p! This issue is on my radar now and I'm happy to provide the design support as needed :)

Am I understanding this correctly, that we're after a notice message, as described in [this issue] ?(https://github.com/Automattic/woocommerce-payments/issues/6294#issuecomment-1944434429)

More details will be appreciated :)

**Comment by haszari** [comment]
> Am I understanding this correctly, that we're after a notice message, as described in [this issue] ?(https://github.com/Automattic/woocommerce-payments/issues/6294#issuecomment-1944434429)

Yes – that comment and the description are a good summary.

TLDR:

- Payouts can fail for various reasons.
- When they fail, we notify merchant by email.
- If merchant doesn't check their email, or is lost/missed, they have no way to know there's a problem.
- ⇒ So we should show something in WooPayments to give them a heads up.

What more details are you looking for @rogermattic ? Maybe we can discuss in slack.

**Comment by rogermattic** [comment]
Thanks @haszari 

>What more details are you looking for @rogermattic ? Maybe we can discuss in slack.

No more details needed for now. I'll start working on this as soon as I can.

**Comment by rogermattic** [comment]
Subject to the final discussion [here](https://a8c.slack.com/archives/C02BW3Z8SHK/p1722964056298259), these are the designs that we can use:

Inline with the Payments Overview notice...

<img width="1500" alt="Payments Overview – Deposit Failure Notice" src="https://github.com/user-attachments/assets/3a59fcbe-0abe-4f04-890e-ebe6099346eb">

[Figma ->](https://www.figma.com/design/q2b4LTEUhyw0y1vKmfEVpx/Deposits?node-id=3057-69516&t=zVajLS4dM2wxqHPp-1)


We should also include the notice in the Account status box (when and if it exists) 

<img width="1440" alt="WCPay Settings -_ Account Status" src="https://github.com/user-attachments/assets/4d9fa706-fbc9-49df-860b-933d3f6d9048">

[Figma ->](https://www.figma.com/design/q2b4LTEUhyw0y1vKmfEVpx/Deposits?node-id=3059-27608&t=zVajLS4dM2wxqHPp-1)

and in the Payout Box:

<img width="1440" alt="WCPay Settings -_ Deposits" src="https://github.com/user-attachments/assets/fac3007e-23b5-4985-9293-b57cfea7dd0a">

Figma -> [
](https://www.figma.com/design/q2b4LTEUhyw0y1vKmfEVpx/Deposits?node-id=3059-26618&t=zVajLS4dM2wxqHPp-1)

Bonus: In the future, Payouts will have it's own card, so we will have the notice displayed there, and more or less it will look like this:

![Payouts - Future state](https://github.com/user-attachments/assets/8b06b87f-a9c4-4049-ae1a-2abeb3448794)
